csharp-basics/exercises/ClassesAndObjects/Exercise5/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Exercise5
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
```

Comment:
Kas ir šis?

Reply:
Izdzēsu lieko failu.

Reply:
Es šim uzdevumam izveidoju jaunu klasi Date. Program.cs failu atstāju. Ja es izdzēšu Program.cs, tad nevaru vairs uztaisīt "build" un palaist testus, konsoli. 
Kā tu iessaki rīkoties?
Jaunājā "review"es tstāju Program.cs